File: sdks/python/src/boxsdk/models/shared/storagepolicies.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from boxsdk import utils
from ..shared import storagepolicy as shared_storagepolicy


@dataclass_json
@dataclasses.dataclass
class StoragePolicies:
    r"""StoragePolicies
    The part of an API response that describes marker
    based pagination
    """
    
    entries: Optional[list[shared_storagepolicy.StoragePolicy]] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('entries') }})
    limit: Optional[int] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('limit') }})
    next_marker: Optional[int] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('next_marker') }})
    prev_marker: Optional[int] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('prev_marker') }})
    

File: go-sdk/pkg/models/shared/storagepolicies.go
package shared

// StoragePolicies
// The part of an API response that describes marker
// based pagination
type StoragePolicies struct {
	Entries    []StoragePolicy `json:"entries,omitempty"`
	Limit      *int64          `json:"limit,omitempty"`
	NextMarker *int64          `json:"next_marker,omitempty"`
	PrevMarker *int64          `json:"prev_marker,omitempty"`
}
